app/controllers/v0/pension_claims_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'pension_21p527ez/tag_sentry'
```

Comment:
this should not be necessary.  i think there is some 'magic' naming convention setup to automatically load modules in ruby (would need to investigate that)